packages/frontend/src/testing/__mocks__/modules/ReactMarkdwnPreview.mock.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC } from 'react';
+
+type Props = {
+    source: string;
+};
+
+const ReactMarkdownPreview: FC<Props> = ({ source }) => {
+    return <>{source}</>;
+};
```

Comment:
markdown preview mock is simple. just takes whatever and renders it without converting it to markdown. so if you search for an error message in a toaster - it should work!